src/multi-level-checkbox-editor/multi-level-checkbox-editor.less

```diff
@@ -25,20 +26,26 @@
         }
 
         label {
-            padding-left: 8px;
             font-weight: normal;
         }
 
         .header {
             display: flex;
-            align-items: center;
+            align-items: flex-start;
+            justify-content: center;
             flex-direction: column;
-            padding: 10px;
+            padding: 10px 15px;
             border: @default-border;
+            border-bottom: 0;
+            background: @white;
 
             .header-label {
                 margin: 0;
             }
+
+            label {
```

Comment:
I'd prefer a class

Reply:
That would mean that we need to either allow Checkbox component to accept className as prop and then show It on label, or declare class directly on label in checkbox component.
Are you ok with one of those options ?